polyfill/lib/calendar.mjs

```diff
@@ -1144,9 +1134,8 @@ const helperHebrew = ObjectAssign({}, nonIsoHelperBase, {
             if (overflow === 'reject') {
               throw new RangeError(`Hebrew monthCode M05L is invalid in year ${year} which is not a leap year`);
             } else {
-              // constrain to last day of previous month (Av)
+              // constrain to same day of previous month (Shevat)
```

Comment:
Hmm, are we sure that this is the correct behavior?  I thought the clamping behavior was supposed to return the closest day to the desired non-existent month/day, but this isn't the closest.

Reply:
Based on our discussion from Friday, I'm assuming we now both agree that the clamping behaviour should be calendar-specific, and that for the Hebrew calendar it makes sense to clamp to the following month, Adar?

Reply:
Yes, for Hebrew same day in Adar sounds like the correct behavior.